releasenotes/notes/prepare-0.25-2efd7230b0ae0719.yaml

```diff
@@ -0,0 +1,12 @@
+---
+prelude: |
+  The Qiskit Terra 0.25.0 release highlights are:
+
+  * Control-flow operations are now supported through the transpiler at
+    all optimization levels, including levels 2 and 3 (e.g. calling
+    :func:`.transpile` or :func:`.generate_preset_pass_manager` with
+    keyword argument ``optimization_level`` specified as 2 or 3 is now
+    supported).
+
```

Comment:
The other thing that I just remembered that's big enough that we might want to call out in the prelude is the deprecation of `qiskit.algorithms`.